### PR TITLE
Fixed issue of custom filters in datagrid

### PR DESCRIPTION
**Pull request by akate1**
**Description:**
- Added support for multi select custom filter in datagrid
-  Fixed issue of not able to preserve filtered value in custom filter

**JIRA:**
- https://jira.cec.lab.emc.com/browse/OBSDEF-4529

**Testing:**
customFilterIssue

### CODE REVIEW

File: src/datagrid/CustomFilterMulti.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            },
+            onChange && onChange(values),
+        );
+    };
+
+    render() {
+        const {values} = this.state;
+
+        return (
+            <div style={{width: "130px", padding: "1em"}}>
```
Comment (maheshkulkarni4) [inline src/datagrid/CustomFilterMulti.tsx:62]: Is this test code ?  labels are hard coded.  Check if thi need to be moved to stories.

Comment (maheshkulkarni4) [inline src/datagrid/CustomFilterMulti.tsx:62]: Convert inline styles to classes

**Review by maheshkulkarni4 — APPROVED** [review]
LGTM

File: src/datagrid/CustomFilterMulti.tsx (hunk 1 above)
Comment (akate1) [reply src/datagrid/CustomFilterMulti.tsx:62]: @maheshkulkarni4 Can't do this in clarity-react for now. Also, this code is just for story/demo so keeping it as it is

Comment (akate1) [reply src/datagrid/CustomFilterMulti.tsx:62]: @maheshkulkarni4 Can't do this in clarity-react for now. Also, this code is just for story/demo so keeping it as it is

**Pull request merged**